bin/cml/send-comment.js

```diff
@@ -27,6 +25,25 @@ exports.builder = (yargs) =>
         default: 'HEAD',
         description: 'Commit SHA linked to this comment'
       },
+      publish: {
+        type: 'boolean',
+        description:
+          'Upload local files and images linked from the Markdown report'
+      },
+      watch: {
+        type: 'boolean',
+        description: 'Watch for changes and automatically update the report'
+      },
+      triggerFile: {
+        type: 'string',
+        description: 'File used to trigger the watcher',
```

Comment:
Trigger how?

```suggestion
        description: 'If specified, --watch will trigger only when the lockfile is present, and will delete the lockfile',
```

Reply:
Exactly as stated in the “using a trigger file” section on https://github.com/iterative/cml/pull/1026#issue-1250268830

Reply:
Need to do #762 before rewording this again :)

Reply:
update: If you'd prefer to do #762 immediately after this PR please do feel free to resolve this @0x2b3bfa0

Reply:
If you're happy to see https://github.com/iterative/cml/pull/1073 merged immediately after this pull request, fine by me. I would push a new commit to https://github.com/iterative/cml/pull/1073 with unified periods and (perhaps) some better descriptions.

Reply:
resolving unresolved